src/languages/es.ts

```diff
@@ -1374,6 +1379,20 @@ export default {
         isControllingOfficer: 'Estoy autorizado a utilizar la cuenta bancaria de mi compañía para gastos de empresa',
         isControllingOfficerError: 'Debe ser un oficial controlador con autorización para operar la cuenta bancaria de la compañía',
     },
+    // TODO get translation
+    personalInfoStep: {
+        personalInfo: 'Personal info',
```

Comment:
personalInfo: 'Información Personal'
enterYourLegalFirstAndLast: 'Ingrese su Nombre y Apellido'
legalFirstName: 'Nombre'
legalLastName: 'Apellido'
enterYourDateOfBirth: 'Ingrese su fecha de Cumple años'
enterTheLast4: 'Ingrese los últimos 4 dígitos de su NSS'
dontWorry: 'No se preocupe, no hacemos ninguna verificación de créditos'
last4ssn: 'Últimos 4 dígitos de su Número de Seguro Social'
enterYourAddress: 'Ingrese su dirección'
letsDoubleCheck: 'Revisemos que todo esté bien'
byAddingThisBankAccount: 'Agregando esta cuenta bancaria, confirmas que as leído, entendido y aceptado'

Reply:
I was about to ping some1 from Expenisfy for this. Thaanks :D